src/index.tsx

```diff
@@ -2,15 +2,21 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import "./index.css";
 import App from "./App.tsx";
-// import reportWebVitals from "./reportWebVitals.tsx";
+// import reportWebVitals from './reportWebVitals';
+import { ClerkProvider } from "@clerk/clerk-react";
+
+const clerkPubKey =
+  "pk_test_Y2hhbXBpb24tZmVycmV0LTEyLmNsZXJrLmFjY291bnRzLmRldiQ";
```

Comment:
@N-i-z Let's  create an `.env` variable for it! We should not hard code secret keys in a project...